### PR TITLE
OPF docs

### PR DESCRIPTION
**Pull request by Matejk00**
Closes: CXSPA-4778

### CODE REVIEW

**Review by FollowTheFlo — APPROVED** [review]
it looks clear and complete to me.

File: _pages/dev/extending-checkout.md [hunk 1]
```diff
@@ -394,6 +394,51 @@ provideConfig({
 
 **Note:** You can use this same approach to combine all the steps into a single-step checkout.
 
+## Multiple Checkout Flows
+
```
Comment (gladius-mtl) [inline _pages/dev/extending-checkout.md:398]: Does the OPF library need to be installed to enable multiple checkout flows?

File: _pages/dev/extending-checkout.md [hunk 2]
```diff
@@ -394,6 +394,51 @@ provideConfig({
...
+Starting from version [x.x] a new checkout property named "flows" has been introduced. This enhancement facilitates greater flexibility and dynamism in customizing the checkout process based on specific criteria returned from the backend.
+
```
Comment (gladius-mtl) [inline _pages/dev/extending-checkout.md:400]: Is a specific version of the back end required for this work? And if so, do we know which version that is?

File: _pages/install/integrations/open-payment-framework-integration.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+- This will become a table of contents (this text will be scrapped).
+{:toc}
+
+***
+
+## Requirements
+
+To integrate SAP Open Payment Framework with Composable Storefront, you must have SAP Commerce Cloud 2211 or newer.
+
```
Comment (gladius-mtl) [inline _pages/install/integrations/open-payment-framework-integration.md:31]: Do we know which version of 2211 is required?

File: _pages/install/integrations/open-payment-framework-integration.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+#### Support for 3DS
+
+It is important to note that all the above UX patterns support 3DS (3-D Secure) authentication. This means that consumers will benefit from an additional layer of security during the payment process, regardless of which UX pattern they encounter.
+
+## Backend Configuration
+
+### Sample data
+
+Certain features of OPF, including the payment and review page or the call-to-action scripts, necessitate specific sample data configurations on the backend.
+
```
Comment (gladius-mtl) [inline _pages/install/integrations/open-payment-framework-integration.md:56]: Can we (should we) provide more information about what these "specific sample data configurations" are?

File: _pages/install/integrations/open-payment-framework-integration.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+If the OPF features do not function correctly, import the following impex script into the backend system:
+
```
Comment (gladius-mtl) [inline _pages/install/integrations/open-payment-framework-integration.md:58]: Here, we're saying "if the OPF features do not function correctly", but at this point in the documentation, the opf lib hasn't been installed yet... Is this information that should go in a troubleshooting section later in the doc, or do customers always need to run this impex for opf to work, and I should be just rephrasing this here to say, you need to run this impex for your OPF features to work properly?....... 🙂

File: _pages/dev/extending-checkout.md (hunk 1 above)
Comment (Matejk00) [reply _pages/dev/extending-checkout.md:398]: No, multiple checkout flows strategy can be enabled even if OPF library has not been installed.

File: _pages/dev/extending-checkout.md (hunk 2 above)
Comment (Matejk00) [reply _pages/dev/extending-checkout.md:400]: There is no any specific backend version required for it.

File: _pages/install/integrations/open-payment-framework-integration.md (hunk 3 above)
Comment (Matejk00) [reply _pages/install/integrations/open-payment-framework-integration.md:31]: I cannot specify the exact version required at this moment. Could we perhaps insert a placeholder here and revisit it closer to the OPF release date?

File: _pages/install/integrations/open-payment-framework-integration.md (hunk 4 above)
Comment (Matejk00) [reply _pages/install/integrations/open-payment-framework-integration.md:56]: Nah, that's a wrong wording I believe. It shouldn't be "configurations" there. Just "necessitate specific sample data on the backend.".

File: _pages/install/integrations/open-payment-framework-integration.md (hunk 5 above)
Comment (Matejk00) [reply _pages/install/integrations/open-payment-framework-integration.md:58]: Exactly, customers must execute this impex for OPF to operate correctly. If they encounter issues after installing the OPF library and find certain functionalities missing, they should verify that they've imported the necessary impex scripts.

File: _pages/dev/extending-checkout.md [hunk 6]
```diff
@@ -394,6 +394,55 @@ provideConfig({
...
+**Note:** Multiple checkout flows requires Spartacus 2211.x
+
+### Configuring Multiple Checkout Flows
+
+The `flows` checkout out property allows you to inject different checkout flows using specific keys. When the checkout process is initiated, the checkout orchestrator service queries the back end for the `paymentProvider` value. You can set a separate `paymentProvider` value for each base store. If the returned `paymentProvider` value matches a predefined checkout flow key, that specific flow is served to the customer. For example, if the back end returns a value of `ProviderA` for the `paymentProvider`, and there is a corresponding checkout flow with the `ProviderA` key, then the `ProviderA` checkout flow is activated.
+
+**Note:** Using multiple checkout flows in optional. If a specific checkout flow key is not found, the service falls back to the standard checkout steps and configuration.
+
+The following is an example of a `checkout.flows` configuration:
+
```
Comment (gladius-mtl) [inline _pages/dev/extending-checkout.md:410]: I'm not sure if `checkout.flows` is the correct annotation for referring to `checkout : { flows: { ... `
If not, I can probably just change it to "`flows` configuration".

File: _pages/install/integrations/open-payment-framework-integration.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+  },
+}),
+```
+
+The following is a description of the Commerce Cloud Adapter configuration properties:
+
+- `baseUrl` is the URL to the Commerce Cloud Adapter. This URL has the same domain as your workbench URL, which is provided to you by an SAP administrator. For more information, contact your SAP administrator for Open Payment Framework.
+
+- `commerceCloudPublicKey` is the public key provided by Open Payment Framework. This key is used to establish a connection between the storefront and the instance of SAP Commerce Cloud that provided this key. For more information about generating the `commerceCloudPublicKey`, see [Configure Client in Open Payment Framework Workbench](https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/0996ba68e5794b8ab51db8d25d4c9f8a/abd0dcd86a5d472e8fd6d22bff28e9c4.html?locale=en-US&state=DRAFT).
+
```
Comment (gladius-mtl) [inline _pages/install/integrations/open-payment-framework-integration.md:104]: I changed the wording (and meaning) somewhat of what is says here about `commerceCloudPublicKey`. I don't know if this is actually correct and/or useful information, but if I have gone too far in a wrong direction, just let me know! 😄

File: _pages/dev/extending-checkout.md [hunk 8]
```diff
@@ -394,6 +394,55 @@ provideConfig({
...
+Multiple checkout flows allows you to provide realtime switching between checkout flows, and eliminates the need to rebuild your storefront app for each change. This allows you to provide multiple checkout flows to cater to different market segments or product types.
+
+**Note:** Multiple checkout flows requires Spartacus 2211.x
+
+### Configuring Multiple Checkout Flows
+
+The `flows` checkout out property allows you to inject different checkout flows using specific keys. When the checkout process is initiated, the checkout orchestrator service queries the back end for the `paymentProvider` value. You can set a separate `paymentProvider` value for each base store. If the returned `paymentProvider` value matches a predefined checkout flow key, that specific flow is served to the customer. For example, if the back end returns a value of `ProviderA` for the `paymentProvider`, and there is a corresponding checkout flow with the `ProviderA` key, then the `ProviderA` checkout flow is activated.
+
+**Note:** Using multiple checkout flows in optional. If a specific checkout flow key is not found, the service falls back to the standard checkout steps and configuration.
```
Comment (Matejk00) [inline _pages/dev/extending-checkout.md:407]: Could you confirm that there is no typo here, do you mean "is optional"? I'm not sure if "in optional" is a correct version.

File: _pages/dev/extending-checkout.md (hunk 6 above)
Comment (Matejk00) [reply _pages/dev/extending-checkout.md:410]: In theory it's correct, as it refers to object hierarchy. But, I would prefer just to use "`flows` configuration" in this case.

File: _pages/install/integrations/open-payment-framework-integration.md (hunk 7 above)
Comment (Matejk00) [reply _pages/install/integrations/open-payment-framework-integration.md:104]: Correct, this version is more informative than a previous one. We can keep it for sure.

Comment (gladius-mtl) [reply _pages/install/integrations/open-payment-framework-integration.md:104]: Perfect! Thank you very much 😊

File: _pages/dev/extending-checkout.md (hunk 8 above)
Comment (gladius-mtl) [reply _pages/dev/extending-checkout.md:407]: Definitely a typo! Thank you catching that! 😄

File: _pages/dev/extending-checkout.md (hunk 6 above)
Comment (gladius-mtl) [reply _pages/dev/extending-checkout.md:410]: Perfect! 🙂

**Review by gladius-mtl — APPROVED** [review]

**Pull request merged**